docs/extensibility/resources.md

```diff
@@ -0,0 +1,224 @@
+
+#### Item paremeters
+
+- **path** - path for this resource used in the url.
+- **scope** - either `namespace` or `cluster`.
+
+#### Example
+
+```json
+{
+  "path": "my-resources",
+  "scope": "namespace"
+}
+```
+
+### form section
+
+The `form` section modifies the original schema (provided in the schema section) for use with [ui-schema](https://ui-schema.bemit.codes/) to generate the create/edit form for the resource. It contains a list of objects that define which fields must be included in the final form. All given fields are placed in the advanced form by default. It's possible to add a field to simple form by providing the `simple: true` flag, or removing it from advanced form by providing the `advanced: false` flag.
+
+#### Item parameters
+
+- **path** - _[required]_ path to the property that must be displayed in the form. In case of an array the array index is omitted. For example, if `spec.items` is an array and you want to display `name` for each items, the path is `spec.items.name`).
+- **widget** - optional widget to render the field. If no widget is provided a default handler is used depending on the data type provided in the schema. For more information about the available widgets, see [Form widgets](form-widgets.md).
+- **simple** - to display in the simple form. By default it is false.
+- **advanced** - to display in the advanced form. By default it is true.
+
+### Example
+
+```json
+[
+  { "path": "spec.priority", "simple": true },
+  { "path": "spec.items.name" },
+  { "path": "spec.items.service.url" },
+  { "path": "spec.items.service.port" }
+]
+```
+
+### list section
+
+The `list` section defines extra columns available in the list. The format is similar to the form section, however each entry consists only of two values:
+
+#### Item parameters
+
+- **path** - _[required]_ contains the path to the data used for the column.
+- **widget** - optional widget used to render the field. By default the value is displayed verbatim. For more information about the available widgets, see [Display widgets](display-widgets.md).
+
+#### Example
+
+```json
+[{ "path": "spec.url" }, { "path": "spec.priority", "widget": "Badge" }]
+```
+
+### details section
+
+The `details` section defines the display structure for the details page. It contains two sections, `header` and `body`, both of which are a list of item to display in the header section and the body of the page respectively. The format of the entries is similar to the form section, however it has extra options available.
+
+#### Items parameteres
+
+- **path** - contains the path to the data used for the widget. Not required for purely presentational widgets.
+- **name** - used for entries without `path` to define the translation source used for labels. Required if no path is present.
+- **widget** - optional widget to render the field. By default the value is displayed verbatim. For more information about the available widgets, see [Display widgets](display-widgets.md).
```

Comment:
Can you provide more information about the type of field this parameter renders?